packages/contractkit/src/utils/phone-number-lookup/phone-number-lookup.ts

```diff
@@ -34,6 +34,7 @@ export interface PhoneNumberPrivacyRequest {
 
 export interface SignMessageRequest extends PhoneNumberPrivacyRequest {
   blindedQueryPhoneNumber: string
+  timestamp: number
```

Comment:
If this isn't made optional, i think this will break the mobile app which currently doesn't include it. You'd have to update that too or make it optional for now.

Reply:
Good call out. I think we should make it optional to make it backward compatible with the existing mobile apps in the wild.

Reply:
If the mobile app is using these changes won't it also include the change above in `phone-number-identifier.ts` that sets` timestamp: Date.now() ? `

Reply:
I agree its probably better to have this optional just trying to understand everything fully

Reply:
@alecps ah I missed that bit in `phone-number-identifier.ts`. In that case we should be fine so long as the combiner/signers keep it optional.